src/cli/client.rs

```diff
@@ -171,6 +171,10 @@ impl KrillClient {
                 post_empty(&self.server, &self.token, "api/v1/bulk/cas/suspend").await?;
                 Ok(ApiResponse::Empty)
             }
+            BulkCaCommand::Import(structure) => {
+                post_json(&self.server, &self.token, "api/v1/bulk/cas/import", structure).await?;
+                Ok(ApiResponse::Empty)
```

Comment:
Minor observation: as all branches end with `Ok(ApiResponse::Empty)` you could factor that out and as a bonus make it clearer at a glance that all bulk operations yield an empty response.

Reply:
sure. Done